Smith_Ben/Assignments/email_validation/emai.py

```diff
@@ -0,0 +1,57 @@
+from flask import Flask, request, redirect, render_template, session, flash
+from mysqlconnection import MySQLConnector
+import re
+app = Flask(__name__)
+mysql = MySQLConnector(app, 'emailval')
+app.secret_key = 'secret'
+EMAIL_REGEX = re.compile(r'^[a-zA-Z0-9.+_-]+@[a-zA-Z0-9._-]+\.[a-zA-Z]+$')
+
+@app.route('/')
+def validation():
+	return render_template('validation.html')
+
+
+@app.route('/emails', methods=['POST'])
+def email():
+   if not EMAIL_REGEX.match(request.form['buttonbox']):
+   	flash('invalid emale')
+   	return redirect('/')
+   else:
+   	flash ('Great Job!');
+
+   query = "INSERT INTO email (email,updated_at,created_at) VALUES (:email,NOW(),NOW())"
+   data = {'email':request.form['buttonbox']}
+   mysql.query_db(query,data)
+   query = "SELECT created_at FROM email"
+   query = "SELECT * FROM email"
+   email = mysql.query_db(query)
+   # if len(request.form['buttonbox']) < 1:
+   # 	flash('need a proper emale')
+
+
+   return render_template('email.html', email = email)
```

Comment:
Is this a `render_template` I see on a `methods = ['POST']` ?

What could possibly be wrong with this?!?!?!?!